### PR TITLE
Adds large metal boxes: 2015 Unmerged Pull of the Year Edition

### PR DESCRIPTION
**Pull request by Incoming5643**
Like the normal ones, except:

*can fit five people
*can take a lot of punishment before breaking
*slow as garbage to move in
*only exist near the syndicate drop pods
*became utterly required once STEEL RAIN was added

Literally a remake of #14053 
I don't need a token because this predates the toke system.

:cl: incoming5643
rscadd: Nuke ops can now harness the power of METAL BOXES: special versions of cardboard boxes that can take a lot of abuse and can fit the whole team if need be. They're not nearly as light as their cardboard brothers though...
/:cl:

### CODE REVIEW

**Comment by Incoming5643** [comment]
Maps shuffled around some defines unrelated to this pull?

Why? No idea.

Cannot stop it.

**Comment by kilkun** [comment]
MERGE THIS!!!!

File: code/game/objects/structures/crates_lockers/closets/cardboardbox.dm [hunk 1]
```diff
@@ -19,7 +20,7 @@
 		return
 	move_delay = 1
 	if(step(src, direction))
-		spawn(config.walk_speed)
+		spawn(config.walk_speed*move_speed_multiplier)
```
Comment (optimumtact) [inline code/game/objects/structures/crates_lockers/closets/cardboardbox.dm]: I am not a fan of this spawn, but it's hardly your fault

Comment (optimumtact) [inline code/game/objects/structures/crates_lockers/closets/cardboardbox.dm]: although assuing relay_move gets updated every time, you could change this to a

    next_move = worldtime+move_delay
    if(next_move > worldtime) 
        return

Comment (Incoming5643) [inline code/game/objects/structures/crates_lockers/closets/cardboardbox.dm]: but rollover protection

Comment (optimumtact) [inline code/game/objects/structures/crates_lockers/closets/cardboardbox.dm]: as discussed in irc I believe that applies to world.timeofday, not world.time

**Comment by xxalpha** [comment]
Please make sure you have the up to date versions of maps before you edit them.

**Comment by Incoming5643** [comment]
Pull was made from a copy of master less than one hour older than this pull request.

I think perhaps any z2 map edit might have those changes until one of them gets merged

**Comment by xxalpha** [comment]
Someone fucked up somewhere and z2 is now fucked or something. Coiax had a fixed version.

**Comment by kilkun** [comment]
>500 health
>A lot of punishment
2.5 laser guns isn't a lot of punishment if you ask me.

File: code/game/objects/structures/crates_lockers/closets/cardboardbox.dm (hunk 1 above)
Comment (kilkun) [inline code/game/objects/structures/crates_lockers/closets/cardboardbox.dm]: that makes it less readable, though.

Comment (optimumtact) [inline code/game/objects/structures/crates_lockers/closets/cardboardbox.dm]: I don't care, spawn is expensive

**Comment by Jazaen** [comment]
Since when do laser guns do 200 damage?

**Comment by Iamgoofball** [comment]
thats half the armory's laserguns and a bit extra

**Comment by optimumtact** [comment]
Still no picture

**Comment by optimumtact** [comment]
Day 2, there is still no picture, my supplies are running low. Hope dwindles

**Comment by as334** [comment]
Sprites @optimumtact 

Open
Closed

**Comment by xxalpha** [comment]
Those are just gray cardboard boxes.

**Comment by optimumtact** [comment]
Day 3, I had almost given up hope, until visited by a disciple, he granted unto me the sprites. At last my sojourn was at an end

**Comment by Incoming5643** [comment]
I was actually going to try and sprite a new box with a little rhino head logo on it, but I didn't get around to it.

Maybe later though

**Pull request merged**